### DISCUSSION TITLE
Make the admin panel only accessible from the local network in wan mode

### DISCUSSION THREAD

**Discussion by Nevenit** [Ylianst/MeshCentral#6566] [General]
Basically what i want is being able to access the web interface only from the local network but still have remote agents. I tried looking around if its possible but i didnt find anything, also the discord link is expired.

**Comment by si458** [comment]
![Screenshot_20241128_100930_Firefox](https://github.com/user-attachments/assets/222025ed-625e-41e5-a1ce-f0a5f9f4d027)
I believe these are the 2 options u can use inside ur `domains` in config.json :)

Also thanks will update discord link!

**Reply by si458** [reply]
The discord link is still showing as valid?
https://discord.gg/8wHC6ASWAc

**Reply by Nevenit** [reply]
It definitively works now. I tried it multiple times yesterday and it said that it was invalid.

**Reply by si458** [reply]
> It definitively works now. I tried it multiple times yesterday and it said that it was invalid.

Maybe discord was having problems 😅
